salt/orchestrate/aws/s3_buckets/xpro.sls

```diff
@@ -1,5 +1,3 @@
-{% for env in ['mitxpro-qa', 'mitxpro-production'] %}
```

Comment:
What does this file have to do with bootcamps?

Reply:
Nothing, aside from the fact that I used it as a template for the code to create bootcamp buckets and I saw the bug while I was in there.